docs/index.bs

```diff
@@ -201,20 +201,21 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
           * <dfn export id="dfn-functional-events">Functional events</dfn>: {{fetch!!event}} and the [=events=] defined by other specifications that <a href="#extensibility">extend</a> the Service Workers specification. (See the <a href="#execution-context-events">list</a>.)
           * {{message!!event}} and {{messageerror!!event}}.
     </section>
-    <section>
-      <h4 id="service-worker-timing">Timing</h4>
+  </section>
 
-      Service workers mark certain points in time that are later exposed by the {{PerformanceNavigationTiming|navigation timing}} API.
+  <section>
+    <h3 id="service-worker-timing">Serview Worker Timing</h3>
 
-      A <dfn export>service worker timing info</dfn> is a [=/struct=]. It has the following [=struct/items=]:
+    Service workers mark certain points in time that are later exposed by the {{PerformanceNavigationTiming|navigation timing}} API.
```

Comment:
```suggestion
    Service workers mark certain points in time that are later exposed by the [=PerformanceNavigationTiming!!idl|navigation timing=] API.
```

I haven't tested if this works, but it should mean that "navigation timing" isn't displayed as if it's a method/object.